src/frontend/src/optimizer/plan_node/logical_join.rs

```diff
@@ -423,7 +422,10 @@ impl ToStream for LogicalJoin {
             }
         } else {
             // Convert to Nested-loop Join for non-equal joins
-            todo!("nested loop join")
+            Err(RwError::from(ErrorCode::NotImplemented(
+                "nested loop join".to_string(),
+                None.into(),
```

Comment:
Do we have a tracking issue for this? It's better to fill the issue ID here. cc @fuyufjh

Reply:
Created a issue just now #2655